### ISSUE TITLE
Commits are not boxed in live view

### ISSUE THREAD

**Issue by patrickbkr** [lizmat/App-Raku-Log#8]
In the live view (in contrast to viewing a log for a specific date), geths commit announcements aren't put in a block.

**Comment by lizmat** [comment]
Yeah... that's a bit... troublesome... still trying to figure out a way to handle "incomplete" commit messages (or test-t messages for that matter).  Especially when other messages can intersperse them.

**Comment by patrickbkr** [comment]
No worries. Just wanted to mention it so it's known to be known.

Also: Thank you very much for your work on the log archiver and viewer. I am really impressed by the result!

**Comment by lizmat** [comment]
The live view will now apply the boxing of commit and test-t messages, and scrolldown also works correctly now.   Scrollup doesn't do that correctly yet, working on that.
